.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please don't add this or the `.yarn` changes :)

Reply:
(or `package-json.lock`)

Reply:
This file should be deleted as well as the changes to `yarn.lock` and `package.json`.